src/SessionSetup/ConnectToStadiaWidget.cpp

```diff
@@ -92,25 +101,69 @@ ConnectToStadiaWidget::ConnectToStadiaWidget(QWidget* parent)
                    &ConnectToStadiaWidget::UpdateRememberInstance);
   QObject::connect(ui_->refreshButton, &QPushButton::clicked, this,
                    [this]() { emit InstanceReloadRequested(); });
+  QObject::connect(ui_->instancesFilterLineEdit, &QLineEdit::textChanged, &instance_proxy_model_,
+                   &QSortFilterProxyModel::setFilterFixedString);
+  QObject::connect(ui_->refreshButton_2, &QPushButton::clicked, this,
+                   [this]() { emit InstanceReloadRequested(); });
+  QObject::connect(ui_->comboBox, QOverload<int>::of(&QComboBox::activated), this,
+                   &ConnectToStadiaWidget::ProjectComboBoxActivated);
+  QObject::connect(ui_->allInstancesCheckBox, &QCheckBox::stateChanged, this, [this]() {
+    QSettings settings;
+    settings.setValue(kAllInstancesKey, ui_->allInstancesCheckBox->isChecked());
+  });
 
   SetupStateMachine();
 
   // The instance settings (mainly project selection and "all instances") is currently hidden behind
   // the "--enable_project_selection" flag.
   // TODO(b/190670843): Clean this up when removing the "--enable_project_selection" flag.
   if (absl::GetFlag(FLAGS_enable_project_selection)) {
```

Comment:
Could you move this into a separate method? The length of the constructor is getting out of hand.

Reply:
done